crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -92,6 +94,24 @@ impl View3DState {
         space_cameras: &[SpaceCamera3D],
         view_coordinates: Option<ViewCoordinates>,
     ) -> &mut OrbitEye {
+        if !self.did_interact_with_eye {
+            let target = default_eye(scene_bbox_accum, &view_coordinates);
+            // Inlined interpolate_to_orbit_eye since we can't borrow self as mut another time
+            if let Some(start) = self.orbit_eye {
+                let target_time = EyeInterpolation::target_time(&start.to_eye(), &target.to_eye());
+                self.spin = false; // the user wants to move the camera somewhere, so stop spinning
+                self.eye_interpolation = Some(EyeInterpolation {
+                    elapsed_time: 0.0,
+                    target_time,
+                    start,
+                    target_orbit: Some(target),
+                    target_eye: None,
+                });
+            } else {
+                self.orbit_eye = Some(target);
+            }
+        }
```

Comment:
Don't understand the borrow concern. this just works for me
```suggestion
        if !self.did_interact_with_eye {
            let target = default_eye(scene_bbox_accum, &view_coordinates);
            self.interpolate_to_orbit_eye(target);
        }
```